test/gravity.db.sql

```diff
@@ -213,7 +213,7 @@ INSERT INTO adlist VALUES(1,'https://hosts-file.net/ad_servers.txt',1,1559928803
 INSERT INTO gravity VALUES('whitelisted.ftl',1);
 INSERT INTO gravity VALUES('gravity.ftl',1);
 INSERT INTO gravity VALUES('gravity-whitelisted.ftl',1);
-INSERT INTO info VALUES("gravity_count",3);
+INSERT INTO info VALUES('gravity_count',3);
```

Comment:
Changed because of

> As of SQLite 3.27.0 (2019-02-07) the use of a double-quoted string literal causes a warning message to be sent to the error log.

https://www.sqlite.org/quirks.html